atomata/capstone_WebClient
# Serialize experience

### Summary 
This PR adds the functionality to save an experience ( actions sequence) of a given user to the cloud storage. 
Note, the user id and login needs to be implemented for future. for now, it saves the experience with the fixed id "test_exp1" for a fixed user "test_user1".  Each experience has experience_id , apparatus_id and list of actions.  Each user has an experience file where all the experiences for that user will be saved in json format.
### Details
1) added save button to editor page
2) clicking the save button, stores the experience in the cloud storage. If the experience file already exists, it will we overwritten to update the given experience.
### How to test?
1) Run the application
2) add some actions to the action sequence ( reorder, delete, ..)
3) click on save button
4) in the cloud storage, in the experience container there should exist the file named "test_user1.json" which contains the data for "test_experience1".  [If you don't have access to cloud storage, open the inspector and find the console.write that says " "the response value is : " it will show the content of the experience folder. 
5) you can rename the experience id to something else in overlay.ts e.g "test_exp2", then save again. This time in the same file, you should see two experiences with respective info for each. 



### Checks
